content/docs/reactivesearch/vue-searchbox/examples.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Examples'
+meta_title: 'Examples for Vue SearchBox'
+meta_description: 'vue-searchbox is a lightweight library that provides scaffolding to create search experiences powered by Elasticsearch.'
+keywords:
+    - vue-searchbox
+    - search library
+    - elasticsearch
+sidebar: 'docs'
+nestedSidebar: 'vue-searchbox-reactivesearch'
+---
+
+The following are some examples to demonstrate the usge of various props available with `vue-searchbox`.
+
+In examples below there are three components present `search-base`, `search-box` & `search-component`.
+
+#### SearchBase
+
+This component binds the backend app(data source) with the UI components. You can read more about it [here](/docs/reactivesearch/vue-searchbox/searchbase/).
+
+#### SearchBox
+
+This component provides a lightweight, performance focused & customizable searchbox UI. You can read more about it [here](/docs/reactivesearch/vue-searchbox/searchbox/).
+
+#### SearchComponent
+
+This search component can be used to bind to different types of search UI widgets. You can read more about it [here](/docs/reactivesearch/vue-searchbox/searchcomponent/).
+
+### Basic Usage
+
+In this example we can see the usage of some of the props that are available with vue `search-box` like `autoSuggest`, `enablePopularSuggestions`, `enableRecentSearches`, `showVoiceSearch`, usage of `QueryRules` to get the promoted results, etc.
+
+<iframe src="https://codesandbox.io/embed/vue-searchbox-basic-example-forked-0jg7i?fontsize=14&hidenavigation=1&theme=dark"
+     style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+     title="vue-searchbox-basic-example"
+     allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+     sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+   ></iframe>
+
+### Controlled Usage
+
+In this example we can the usage of the transformRequest prop, which gives us the request object whenever a query gets triggered from the `search-box` component, we have then used this request object to extract the search query text and then make a call to an external API (Datamuse) to get correct matching texts in case the search query was misspelled, the suggested texts are shown to the user based upon their matching score. The user can then select a text which will be set to a state variable that's been passed to the `value` prop. The `onChange` is the callback function that accepts the component's current value as a parameter and is then used to set the state varaible to been passed to the `value` prop.
```

Comment:
```suggestion
In this example we used the `transformRequest` prop, which gives us the request object whenever a query gets triggered from the `search-box` component, we have then used this request object to extract the search query text and then make a call to an external API (Datamuse) to get correct matching texts in case the search query was misspelled, the suggested texts are shown to the user based upon their matching score. The user can then select a text which will be set to a state variable that's been passed to the `value` prop. The `onChange` is the callback function that accepts the component's current value as a parameter and is then used to set the state varaible to been passed to the `value` prop.
```